.gitignore

```diff
@@ -26,3 +26,5 @@ pulsar.zip
 /fonts/_config.fonts.scss
 
 /.csscomb.json
+/validation-status.json
```

Comment:
These files will also need adding to the `.gitattributes` file as `export-ignore` rules so that Continuum products can ignore them